kotlin-coroutines-repository/app/src/main/java/com/example/android/kotlincoroutines/main/MainViewModel.kt

```diff
@@ -19,13 +19,8 @@ package com.example.android.kotlincoroutines.main
 import androidx.lifecycle.LiveData
 import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
-import com.example.android.kotlincoroutines.main.TitleRepository.RefreshState.Success
-import com.example.android.kotlincoroutines.main.TitleRepository.RefreshState.Error
-import com.example.android.kotlincoroutines.main.TitleRepository.RefreshState.Loading
+import com.example.android.kotlincoroutines.main.TitleRepository.RefreshState.*
```

Comment:
Nit: Update editor preferences to avoid star imports (they're on by default in Kotlin)

Prefs > Editor > Code Style > Kotlin